docs/src/examples.rst

```diff
@@ -14,205 +14,175 @@
-   import cdk = require('@aws-cdk/cdk');
+There are two different ways of creating a serverless infrastructure with |ECS|:
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+- Using the **Fargate** launch type, where |ECS| manages your cluster resources
+- Using the **EC2** launch type, where you manage your cluster resources
 
-       const vpc = new ec2.VpcNetwork(this, 'VPC');
+This example creates a Fargate service,
+which requires a VPC, a cluster, a task definition, and a security group.
+Later on we'll show you how to launch an EC2 service.
 
-       new rds.DatabaseCluster(this, 'MyRdsDb', {
-         defaultDatabaseName: 'MyAuroraDatabase',
-         masterUser: {
-           username: 'admin',
-           password: '123456'
-         },
-         engine: rds.DatabaseClusterEngine.Aurora,
-         instanceProps: {
-           instanceType: new ec2.InstanceTypePair(ec2.InstanceClass.Burstable2, ec2.InstanceSize.Small),
-           vpc: vpc,
-           vpcPlacement: {
-             subnetsToUse: ec2.SubnetType.Public
-           }
-         }
-       });
-     }
-   }
+.. tabs::
 
-   const app = new cdk.App();
+    .. group-tab:: TypeScript
 
-   new MyStack(app, 'MyStack');
+        Add the following import statements:
 
-   app.run();
+        .. code-block:: typescript
 
-.. _creating_s3_example:
+            import ec2 = require('@aws-cdk/aws-ec2');
+            import ecs = require('@aws-cdk/aws-ecs');
 
-Creating an |S3| Bucket
-=======================
+        Add the following code to the end of the constructor:
 
-The following example creates the |S3| bucket **MyS3Bucket** with server-side KMS
-encryption provided by |S3|.
+        .. code-block:: typescript
 
-.. code-block:: js
+            const vpc = new ec2.VpcNetwork(this, 'MyVpc', {
+              maxAZs: 2 // Default is all AZs in region
+            });
 
-   import s3 = require('@aws-cdk/aws-s3');
-   import cdk = require('@aws-cdk/cdk');
+            // Create an ECS cluster
+            const cluster = new ecs.Cluster(this, 'MyCluster', {
+              vpc: vpc
+            });
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+            const taskDefinition = new ecs.FargateTaskDefinition(this, 'MyFargateTaskDefinition', {
+              cpu: '256',  // Default
+              memoryMiB: '2048'  // Default is 512
+            });
 
-       new s3.Bucket(this, 'MyBucket', {
-         bucketName: 'MyS3Bucket',
-         encryption: s3.BucketEncryption.KmsManaged
-       });
-     }
-   }
+            // The task definition must have at least one container.
+            // Otherwise you cannot synthesize or deploy your app.
+            taskDefinition.addContainer('MyContainer', {
+              image: ecs.ContainerImage.fromDockerHub('MyDockerImage')    // Required
```

Comment:
Maybe launch `amazon/amazon-ecs-sample`, that's a container that at least does something.